front/pages/w/[wId]/members/index.tsx

```diff
@@ -638,11 +636,39 @@ function ChangeMemberModal({
           </Page.P>
         </div>
       </div>
+      <Modal
+        onClose={() => setRevokeMemberModalOpen(false)}
+        isOpen={revokeMemberModalOpen}
+        title="Revoke member access"
+        hasChanged={false}
+      >
+        <div className="mt-6 flex flex-col gap-6 px-2">
+          <div>
+            Revoke access for user{" "}
+            <span className="font-bold">{member.name}</span>?
+          </div>
+          <div className="flex gap-2">
+            <Button
+              variant="tertiary"
+              label="Cancel"
+              onClick={() => setRevokeMemberModalOpen(false)}
+            />
+            <Button
+              variant="primaryWarning"
+              label="Yes, revoke"
+              onClick={() => {
+                handleMemberRoleChange(member, "none");
```

Comment:
Why not `revoked`? That's what we used to set (both in Membership or in Membership invitation depending on wether the user had created an account or not)

Reply:
RoleType does not accept revoked; only none. Setting none was what was done by the previous version of the page

We can decide RoleType should accept revoked but IMHO that's outside the scope of this PR

Reply:
I trust you with the decision!
I think we should still get "revoked" set in the database as before (my main is before you merged the new page PR, I just tested and we store revoked on the database).

Reply:
Ok I think we might have found a preexisting inconsistency here
I just checked and the member page, before I started my work on it, sent a RoleType (which does not has revoked, but has none) to the membership API -- while the membership API expected a revoked, not a none.  
I'll continue to investigate